code/script.js

```diff
@@ -1,9 +1,100 @@
 // A function that adds and remove the class "active" on the section you click on.
 // We haven't really talked about ´this´ yet, but we will... ;)
 // console.log(this) to see how it works
-function toggle() {
-  this.classList.toggle("active")
-}
+// function toggle() {
+//   this.classList.toggle("active")
+// }
 
 // Selects an HTML element, and calls a function which will be executed when the element is clicked.
-document.getElementById("section1").onclick = toggle
+// document.getElementById("section1").onclick = toggle
+
+
+// Sign in form validation
+const firstName = document.getElementById('firstName')
+const lastName = document.getElementById('lastName')
+const email = document.getElementById('email')
+const message = document.getElementById('message')
+const errorElement = document.getElementById('error')
```

Comment:
Really nice to display the errors/validation messages this way - you seem to understand how to use the DOM in JavaScript already! 🙌